app/controllers/admin/users/gdpr_delete_requests_controller.rb

```diff
@@ -0,0 +1,17 @@
+module Admin
+  module Users
+    class GdprDeleteRequestsController < Admin::ApplicationController
+      layout "admin"
+
+      def index
+        @gdpr_delete_requests = ::Users::GdprDeleteRequest.order("created_at desc").page(params[:page]).per(50)
```

Comment:
We moved away from strings for AR as much as possible:

```suggestion
        @gdpr_delete_requests = ::Users::GdprDeleteRequest.order(created_at: :desc).page(params[:page]).per(50)
```